staging/src/k8s.io/api/core/v1/generated.proto

```diff
@@ -4354,6 +4354,12 @@ message ServiceSpec {
   // sessionAffinityConfig contains the configurations of session affinity.
   // +optional
   optional SessionAffinityConfig sessionAffinityConfig = 14;
+
```

Comment:
Please linewrap comments.

Minor doc/grammar fixes:

topologyKeys is a preference-order list of topology keys which clients should respect when accessing this Service.  If any ready backends exist for index [0], they should always be chosen; only if no backends exist for index [0] should backends for index [1] be considered. If this field is specified, but all entries have no backends that match the topology of the client, the service has no backends for that client and connections should fail.   The special value "" may be used to means "any node".  This catch-all value, if used, only makes sense as the last value in the list.

Reply:
Merge this old comment with the one I just wrote, please :)